src/configurations/destinations/zoho_dev/db-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "ZOHO_DEV",
+  "displayName": "ZOHO_DEV",
+  "config": {
+    "supportedAccountDefinitions": {
+      "rudderAccountId": ["DESTINATION_ZOH__DEV_OAUTH"]
+    },
+    "cdkV2Enabled": true,
+    "supportsVisualMapper": true,
+    "auth": {
+      "type": "OAuth"
+    },
+    "transformAtV1": "router",
+    "saveDestinationResponse": true,
+    "excludeKeys": [],
+    "supportedSourceTypes": ["warehouse","cloud"],
+    "supportedMessageTypes": {
+      "cloud": ["record"]
+    },
+    "syncBehaviours": ["mirror"],
+    "disableJsonMapper": true,
+    "supportedConnectionModes": {
+      "warehouse": ["cloud"]
+    },
+    "includeKeys": ["oneTrustCookieCategories", "ketchConsentPurposes", "consentManagement"],
+    "destConfig": {
+      "defaultConfig": [
+        "rudderAccountId",
+        "region",
+        "trigger",
```

Comment:
we moved all these to connection config, so no need to have these. I think we need to clean up zoho original definition first as it's been a while zoho got migrated to VDM v2 and everything is stable